terraform-sentinel/step4.md

```diff
@@ -0,0 +1,40 @@
+In this step, you will create testing files and run them in the Sentinel CLI
+
+## Create a failing test file
+
+Now that you have created a plan in which your Sentinel policy will fail, you need to create a test file for a failing test.
+
+Open `terraform-sentinel/test/restrict-s3-buckets/fail.json`{{open}} to add a path to the failing mock data.
+
+Copy and paste the relative path to your failing mock in the test file.
+
+```
+"mock": {
+    "tfplan/v2": "../../mock-data/mock-tfplan-fail-v2.sentinel"
+},
+```{{copy}}
+
+Review the rest of the test file. This test ensures the main rule will evalute to false.
+
+## Create a passing test file
+Open `terraform-sentinel/test/restrict-s3-buckets/pass.json`{{open}} and add a path to the passing mock data that has already been provided for you.
+
+Copy and paste the relative path to your passing mock in the test file.
+
+```
+"mock": {
+    "tfplan/v2": "../../mock-data/mock-tfplan-pass-v2.sentinel"
+},
+```{{copy}}
+
+Review the rest of the test file. This test ensures the main rule will evalute to true.
+
+## Run a test in the Sentinel CLI
+
+In your terminal, run a test with the `verbose` flag
+
+```
+sentinel test -verbose restrict-s3-buckets.sentinel
+```{{execute}}
+
+Now that your failing and passing scenarios are detailed in test files, you can upload these to your VCS policy repo and connect it to your Terraform Cloud organization. Your organization must be subscribed to **Team & Governance** in [Terraform Cloud or Terraform Enterprise](https://www.hashicorp.com/blog/announcing-free-trials-for-hashicorp-terraform-cloud-paid-offerings/) to apply your Sentinel policies to your workspaces. 
```

Comment:
Can we interpret the output a little more here? Something like "The test command returned success because the passing test passed and the failing test failed. The output presents the results from each individual block of the passing test, followed by the results of each individual block of the failing test." ... Or something like this. 

What exactly are users uploading here, just the policy? Policy + tests? The whole directory?